damus/Models/HomeModel.swift

```diff
@@ -502,6 +513,26 @@ class HomeModel: ObservableObject {
         }
     }
 
+    func createNotification(displayName: String, conversation: String) {
+        let content = UNMutableNotificationContent()
+        let title = NSLocalizedString("Mentioned by", comment: "Mentioned by heading in local notifcation")
```

Comment:
Thanks for making this localized! Unfortunately, it's not quite right as it assumes that `displayName` always comes after `Mentioned by`, which is not true in all languages.

Also, some nitpicks:
1. Typo in `notifcation` -> `notification`
2. You shouldn't need to wrap `conversation` with quotes. Just use it at face value since it's already a string.

You'll want to do this instead and let the translator decide how to format and translate the text:

```swift
let title = String(format: NSLocalizedString("Mentioned by %@", comment: "Mentioned by heading in local notification"), displayName)
content.title = title
content.body = conversation
```